low-level/sspl_ll_d

```diff
@@ -426,8 +426,9 @@ def _sensors_actuators_factory(sspl_threaded_modules, msgQlist, operating_system
     logger.info("sspl-ll Bootstrap: actuators to load: %s" % (actuators, ))
 
     # Import in the proper classes based upon values from config file
-    if operating_system == OperatingSystem.CENTOS7.value or operating_system == OperatingSystem.RHEL7.value or \
-        operating_system.lower() in OperatingSystem.CENTOS7.value.lower() or operating_system.lower() in OperatingSystem.RHEL7.value.lower():
+    if OperatingSystem.CENTOS7.value in operating_system.lower() or \
```

Comment:
we can make this legacy code better i feel
if operating_system.lower() in OperatingSystem.__members__ 
or something similar

Reply:
Yes, even I thought so, but I couldn't figure out anything better. Trying to figure out something still.

Reply:
thats fine, since this is actual delivery but POC, we are good